articles/quickstart/spa/vuejs/02-calling-an-api.md

```diff
@@ -18,6 +18,10 @@ useCase: quickstart
 
 <!-- markdownlint-disable MD002 MD041 -->
 
+:::note
+For a deep dive into implementing user authentication in Vue, visit the [Vue.js Authentication By Example: Call an API.](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication#integrate-vue-js-with-an-api-server/). This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, and using class components. 
+:::
```

Comment:
Should this mention something like we do in React? People reading this quickstart are not looking into user authentication, but calling an API:

```suggestion
:::note
For a deep dive into making secure calls to an API from Vue, visit the [integrating Vue.js with an API server guide](https://developer.auth0.com/resources/guides/spa/react/basic-authentication#call-a-protected-api-from-react/). This guide provides you with additional details, such setting up a sample Express API server and getting test access tokens from the Auth0 Dashboard.
:::
```